### DISCUSSION TITLE
Understanding the StateViewer Component

### DISCUSSION THREAD

**Discussion by tam11a** [tam11a/rayos-admin-react#18] [Q&A] [answered]
How to use the StateViewer.js in Components?

**Comment by asatter833** [comment, marked as answer]
<StateViewer
          stateList={[
            {
              num: 0,
              title: "Total Order",
            },
            {
              num: 0,
              title: "Total User",
            },

            {
              num: 0,
              title: "order canceled",
            },
          ]}
        />
